<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerDashboardWidget</class>
 <widget class="QWidget" name="CustomerDashboardWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Customer Dashboard</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="customerNameLabel">
     <property name="text">
      <string>Welcome, [Customer]</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="ordersTableView"/>
   </item>
   <item>
    <widget class="QPushButton" name="createOrderButton">
     <property name="text">
      <string>Create New Order</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
